src/gui/shell.cpp

```diff
@@ -87,6 +87,16 @@ Shell::Shell(NeovimConnector *nvim, QWidget *parent)
 		setGuiFont(fontDescription, true /*force*/);
 	}
 
+#if (QT_VERSION > QT_VERSION_CHECK(5, 13, 0))
+	auto screen = this->screen();
+	if (screen) {
+		connect(screen, &QScreen::logicalDotsPerInchChanged,
```

Comment:
Do we still need these two here now that we have the one in showEvent?

Reply:
It fires only on the primary monitor and the font is not adjusted well, so I think it's good to delete it once.